chapters/en/Unit 3 - Vision Transformers/KnowledgeDistillation.ipynb

```diff
@@ -0,0 +1,861 @@
+{
```

Comment:
<p>not sure about choosing temperature = 5, maybe start with 1.0</p><p>also try checking out <a href="https://github.com/lucidrains/vit-pytorch/blob/d446a41243c91a43adcac6f0559d53f1a4eea4fa/vit_pytorch/distill.py#L129" target="_blank">lucidrains/vit_pytorch/distill</a> for the loss implementation. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/shreydan/computer-vision-course/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>1.0 would be not doing anything though, why a temperature of 1.0?</p><p> </p><p>on the loss implementation, it seems mathematically the same as what I used here, no?</p>